cinema/models.py

```diff
@@ -64,7 +64,8 @@ def __str__(self):
 class Order(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     user = models.ForeignKey(
-        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
+        settings.AUTH_USER_MODEL,
+        on_delete=models.CASCADE
     )
 
     def __str__(self):
```

Comment:
add typehint

Reply:
Do we have to add type hints to the models which wasn't written by us?